src/config.js

```diff
@@ -15,13 +15,13 @@ module.exports = {
   MIN_INTRO_MESSAGE_LENGTH: process.env.MIN_INTRO_MESSAGE_LENGTH,
   WELCOME_MESSAGE: `Welcome to the 🌻 💚 **Coding Garden Discord!** 💚 🌻 
 
-Before gaining access to all of the channels, you will need to **read the rules and code of conduct** in the #welcome channel.
+Before gaining access to all of the channels, you will need to **read the rules and code of conduct** in the <#451779463819034626> channel.
```

Comment:
Instead of hard coding the channel ID, can we use the `process.env.WELCOME_CHANNEL_ID` variable?